i18n/german.json

```diff
@@ -557,48 +557,49 @@
     "music-plinfo-warning" : "Warnung : Es gibt einen toten Link in deiner Autoplaylist. Du kannst diesen einfach mit {} entfernen",
     "music-plinfo-too-much" : "Die Autoplaylist hat zu viele Lieder, um hier angezeigt zu werden. Ich werde dir die Playlist in Kürze über den Privat-Chat schicken.",
     "music-plrepair-someone" : "Du kannst nur deine eigene Autoplaylist bearbeiten.",
-    "music-plrepair-loading" : "Ich repariere diese Autoplaylist, bitte gedulte dich einen Moment...",
+    "music-plrepair-loading" : "Ich repariere diese Autoplaylist, bitte gedulde dich einen Moment...",
     "music-plrepair-all-ok" : "Alles ist okay, diese Autoplaylist muss nicht repariert werden!",
     "music-plrepair-removed-links" : "{} tote Links wurden erfolgreich aus der Autoplaylist entfernt.",
     "music-plrepair-removed-link" : "`1` toter Link wurde erfolgreich aus der Autoplaylist entfernt.",
     "music-lyrics-title-not-find" : "Ich kann den Titel nicht finden, bitte vergewissere dich, dass die URL korrekt ist.",
     "music-lyrics-no-current-song" : "Es wird derzeit kein Lied abgespielt. Gib bitte eine URL an.",
     "music-lyrics-not-found" : "Entschuldigung, aber ich konnte die Lyrics zu diesem Lied nicht finden.",
     "music-pladd-already-present" : "Dieses Lied ist bereits in der Autoplaylist!",
-    "music-pladd-added" : "{} wurde zur Auto-Playlist hinzugefügt.",
+    "music-pladd-added" : "{} wurde zur Autoplaylist hinzugefügt.",
     "music-pladd-pl-added" : "Ich habe {} Lieder zu der Autoplaylist hinzugefügt.",
     "music-pladd-pls-added" : "Ich habe {} Lieder zu den schon bestehenden {} hinzugefügt.",
-    "music-pladd-all-already" : "Alle Songs war bereits in der Autoplaylist, ich habe nichts hinzugefügt.",
-    "music-plremove-not-found" : "Ich konnte dieses Lied in der derzeitigen Autoplaylist nicht finden.",
+    "music-pladd-all-already" : "Alle Lieder waren bereits in der Autoplaylist, ich habe nichts hinzugefügt.",
+    "music-pladd-already-present" : "Lied bereits in der Autoplaylist!",   
```

Comment:
This line is already at line 567, ` "music-pladd-already-present" : "Dieses Lied ist bereits in der Autoplaylist!",`
Please remove one of them